client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -52,7 +52,7 @@ class CheckoutThankYouHeader extends PureComponent {
 
 		if ( primaryPurchase && isDomainTransfer( primaryPurchase ) ) {
 			if ( isDelayedDomainTransfer( primaryPurchase ) ) {
-				return preventWidows( translate( 'Congratulations! Your site is live.' ) );
+				return preventWidows( translate( 'Almost done!' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Almost done…' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/transfer-thank-you-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).